spec/classes/puppet_webhook_spec.rb

```diff
@@ -6,6 +6,8 @@
       let(:facts) { os_facts }
 
       it { is_expected.to compile }
```

Comment:
I like to use `compile.with_all_deps`

Reply:
This is generated by PDK. I haven't changed this yet as I don't want failing tests on code that's not done yet (i.e. `puppet_webhook::config`. There shouldn't be any failures, but rather be safe than sorry.

That said, I fully intend to add that, but it'll be closer to initial release than as part of the install class PR.